.github/workflows/build.yml

```diff
@@ -32,3 +32,38 @@ jobs:
           file: ${{github.workspace}}/docker/${{ matrix.ROS_DISTRO }}/Dockerfile
           context: .
           push: false
+
+  build-ros2:
```

Comment:
Why did you add this job? What's the difference between `build` job?

Reply:
Ah, this was added by me? So can you remove this?

Reply:
> Ah, this was added by me? So can you remove this?

Yes, you added this job. So I left it in place.
OK, I will remove this job.

Reply:
de1c6ed

Reply:
Thanks!